bitnami/fluentd/templates/_helpers.tpl

```diff
@@ -134,10 +146,34 @@ fluentd:
 
 {{/* Validate values of Fluentd - must create serviceAccount to create enable RBAC */}}
 {{- define "fluentd.validateValues.rbac" -}}
-{{- if and .Values.rbac.create (not .Values.serviceAccount.create) -}}
+{{- if not (typeIs "<nil>" .Values.rbac.create) -}}
 fluentd: rbac.create
-    A ServiceAccount is required ("rbac.create=true" is set)
-    Please create a ServiceAccount (--set serviceAccount.create=true)
+    Top-level rbac configuration has been removed, as it only applied to the forwarder.
+    Please migrate to forwarder.rbac.create
+{{- end -}}
+{{- if and .Values.forwarder.rbac.create (not .Values.forwarder.serviceAccount.create) -}}
+fluentd: forwarder.rbac.create
+    A ServiceAccount is required ("forwarder.rbac.create=true" is set)
+    Please create a ServiceAccount (--set serviceAccount.forwarder.create=true)
+{{- end -}}
```

Comment:
We should add the same validation for the aggregator, shouldn't we?

Reply:
The `aggregator` doesn't need/have any RBAC - since it didn't have a serviceaccount prior to this change there was no way for the aggregator to have a special `Role`/`ClusterRole`.

The `forwarder` RBAC allows the forwarder to augment the container logs from a node with kubernetes-oriented metadata (namespace, node, node-etc) via the `fluent-plugin-kubernetes_metadata_filter` which doesn't seem like a relevant use case for the aggregator?